src/extensionsIntegrated/Phoenix-live-preview/panel.html

```diff
@@ -3,6 +3,7 @@
         <div style="width: 20%;display: flex;">
             <button id="reloadLivePreviewButton" title="{{clickToReload}}" class="btn-alt-quiet toolbar-button reload-icon"></button>
             <button id="highlightLPButton" title="{{toggleLiveHighlight}}" class="btn-alt-quiet toolbar-button pointer-fill-icon"></button>
+            <button id="wordNavigationLPButton" title="{{toggleWordNavigation}}" class="btn-alt-quiet toolbar-button text-icon"></button>
```

Comment:
We tend to avoid placing new buttons on the live preview unless its absolutely required to keep the UI clutter in check.

Since this can be the default behavior, please tie this to the already existing `toggleLiveHighlight` flag and remove the new button.